UDx-container/README.md

```diff
@@ -111,23 +111,6 @@ Use environment variables to provide additional information to the `vsdk-*` comm
 | VSDK_ENV | Optional file that defines environment variables for `vsdk-*` commands that run in the container. For formatting details, see [Declare default environment variables in file](https://docs.docker.com/compose/env-file/) in the Docker documentation.|
 | VSDK_MOUNT | A list of one or more directories that you want to mount in the UDx container filesystem. To mount multiple directories, separate each path with a space. For additional details, see [Mounting additional files](#mounting-additional-files). |
 
-### Testing the container
-
-The `make test` target calls a few `vsdk-*` scripts to test that your container was built correctly. Then, it mounts the following directories in the UDx container filesystem to replicate your local development environment:
-- `/home/<user-name>`
-- The current working directory and its child directories
-
-For an illustration of the mounted directories, see [Host and container filesystem views](#host-and-container-filesystem-views).
-
-Because the contents of the UDx container are not writable, `make test` calls `vsdk-cp` to copy the `/opt/vertica/sdk/examples` UDx directory into a new directory named `./tmp-test` that is available on your host machine. Next, it builds the examples in that directory with `vsdk-make`.
-
-Run `make test` with the `TARGET` environment variable:
-
-```shell
-$ make test TARGET=deb
-```
-
-
 ### Compiling UDxs
 
 After you [test your UDx container](#testing-the-container), you can develop UDxs in the current working directory on the host machine and compile them in the UDx container. Use the `vsdk-make` script to execute your makefile and compile your UDx. In a new environment, `vsdk-make` mounts the same directories as the `make test` command.
```

Comment:
As the section "Testing the container" was removed, should "[test your UDx container](#testing-the-container)" be changed the tag for "Building the UDx container"?

Reply:
Never mind. I found the "test" section in latter part of the README file.